test/core/crypto/hasher/hasher_test.cpp

```diff
@@ -87,11 +88,8 @@ TEST_F(HasherFixture, sha2_256) {
  * @then expected result obtained
  */
 TEST_F(HasherFixture, blake2_256) {
-  Buffer buffer = {0x69, 0x20, 0x61, 0x6d, 0x20, 0x64, 0x61, 0x74, 0x61};
-  std::vector<uint8_t> match = {0xba, 0x67, 0x33, 0x6e, 0xfd, 0x6a, 0x3d, 0xf3,
-                                0xa7, 0x0e, 0xeb, 0x75, 0x78, 0x60, 0x76, 0x30,
-                                0x36, 0x78, 0x5c, 0x18, 0x2f, 0xf4, 0xcf, 0x58,
-                                0x75, 0x41, 0xa0, 0x06, 0x8d, 0x09, 0xf5, 0xb2};
+  Buffer buffer {"6920616d2064617461"_unhex};
+  std::vector<uint8_t> match = "ba67336efd6a3df3a70eeb757860763036785c182ff4cf587541a0068d09f5b2"_unhex;
```

Comment:
you could make `_unhex_vec` and `_unhex_buf`

Reply:
or `_unhexv`/`_unhexb`

Reply:
Yes, but I think that construction of a Buffer from a vector is simple enough as it is.

Reply:
okai